scripts/backend_tests.py

```diff
@@ -216,6 +221,10 @@ def _convert_to_test_target(path):
                 os.path.join(base_path, root)))
         for subroot, _, files in os.walk(os.path.join(base_path, root)):
             for f in files:
+                if f.endswith('load_test.py') and not exclude_load_tests:
```

Comment:
Ah, no, don't do this. Declare _LOAD_TESTS_DIR = 'core/tests/load_tests', put any load test files in there with more descriptive names, and exclude that dir if needed. We may well have multiple load tests in future for different parts of the site.

Also consider printing a log line to the terminal ("Skipping load test file XXX") when a file is skipped.